src/detect-engine-threshold.c

```diff
@@ -821,7 +823,16 @@ static int ThresholdCheckUpdate(const DetectThresholdData *td, ThresholdEntry *t
                     te->current_count = 1;
                 }
             }
+            if (de_ctx->RateFilterCallback && original_action != pa->action) {
```

Comment:
Is this OK here? Or should it be directly after each call to RateFilterSetAction.

(cd: @amirabell).